Assets/HoloToolkit/Input/Scripts/KeywordManager.cs

```diff
@@ -2,79 +2,97 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System.Collections.Generic;
-using System.Linq;
 using UnityEngine;
-using UnityEngine.Events;
 using UnityEngine.Windows.Speech;
 
 namespace HoloToolkit.Unity
 {
     /// <summary>
-    /// KeywordManager allows you to specify keywords and methods in the Unity
+    /// KeywordManager allows you to specify keywords in the Unity
     /// Inspector, instead of registering them explicitly in code.
     /// This also includes a setting to either automatically start the
     /// keyword recognizer or allow your code to start it.
     ///
     /// IMPORTANT: Please make sure to add the microphone capability in your app, in Unity under
     /// Edit -> Project Settings -> Player -> Settings for Windows Store -> Publishing Settings -> Capabilities
     /// or in your Visual Studio Package.appxmanifest capabilities.
+    /// 
+    /// Please, also make sure to also set your keyword responses in the Inspector window under Keywords and Responses Array.
     /// </summary>
-    public partial class KeywordManager : MonoBehaviour
+    public partial class KeywordManager : Singleton<KeywordManager>
```

Comment:
Also, the scripts placed around the scene would typically only contain keywords relevant to the GameObject they're on. It's still possible to have a high level recognizer for "global" voice commands without preventing an easy implementation of "in-context" voice commands (say, on a single object and turned on on gaze or on a Select).